### ISSUE TITLE
max_threads=1 without verbose output?

### ISSUE THREAD

**Issue by coxley** [knipknap/exscript#72]
Is there a way to use max_threads=1 without the verbose output?

**Comment by egroeper** [comment]
```quickstart(hosts, do_something, verbose=0)```?

**Comment by coxley** [comment]
That worked but I guess it's just not setup to provide the logging if threading isn't being done because that leaves it blank until finish.

**Comment by egroeper** [comment]
You are missing the status bar / progress bar?

**Comment by coxley** [comment]
Yeah and the status messages for as each host begins and ends and the exceptions when errors happen.
